packages/story-editor/src/components/carousel/skeletonPage.js

```diff
@@ -36,8 +39,8 @@ const EmptyPage = styled.li.attrs({ role: 'presentation' })`
   border-radius: 4px;
 `;
 
-function getPatternFromArray([r, g, b]) {
-  return { color: { r, g, b } };
+function getPatternFromArray(baseColor) {
+  return getSolidFromHex(baseColor.replace('#', ''));
 }
```

Comment:
`baseColor` is a string here, why still call the function `...fromArray`? 🤔

Reply:
`generatePatternStyles` used a couple of lines down,  expects a rgba object to be passed it. It is a bigger refactor to have this to expect a string and convert it there.

Reply:
`baseColor` is not an array. So this function should not be called `...fromArray`.

Reply:
Removed.